frontend/src/components/Sidepanel/Panels/Info/Info.js

```diff
@@ -74,7 +74,7 @@ const Info = () => {
             <th>{getStateName(state.id)}</th>
             {alphabet.map(symbol => <td key={symbol}>
               {Object.entries(transitionMap)
-                .filter(([key]) => key.split(',')[0] === state.id && key.split(',')[1] === symbol)
+                .filter(([key]) => key.split(',')[0] === state.id toString() && key.split(',')[1] === symbol)
```

Comment:
.filter(([key]) => key.split(',')[0] === state.id.toString() && key.split(',')[1] === symbol)

this should fix the syntax error, and then ensure linting tests pass even outside this

Reply:
thank you i will try again.